pkg/webhook/conversion/conversion.go

```diff
@@ -69,6 +69,12 @@ func (wh *Webhook) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if convertReview.Request == nil {
+		log.Error(nil, "conversion request is nil")
```

Comment:
I think Error should be used as we're logging an error message here. Also the documentation says "Info logs a non-error message with the given key/value pairs as context" so it may not be appropriate to use Info. What do you think?

Reply:
Error seems okay to me. It definitely seems to fall in the same category as l.67
```go
	if err != nil {
		log.Error(err, "failed to read conversion request")
		w.WriteHeader(http.StatusBadRequest)
		return
	}
```

Reply:
Problem is that we're passing `nil` to `log.Error` which seems a bit backwards, let's either create an error, or spit out an Info as warning for users?

Reply:
Maybe
```go
    log.Error(errors.New("conversion request is nil"), "failed to handle conversion request")
```
?

Stills feels a bit strange, but maybe better than logging one error as an error and the other as info

Reply:
I don't see any issue with err being `nil`, the docs clearly say this is ok (https://pkg.go.dev/github.com/go-logr/logr#Logger.Error and https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md#logging-errors).

Also, there are already many instances of Error being called with a `nil` error in this repo (https://github.com/kubernetes-sigs/controller-runtime/search?q=log.Error%28nil)

Reply:
We went through multiple logger iterations before we got here, but anyways, that's fine if we want to log Error